jsonschema-core.xml

```diff
@@ -3007,142 +3207,185 @@ https://example.com/schemas/common#/$defs/count/minimum
                     </figure>
                 </section>
 
-                <section title="Detailed">
+                <section title="Hierarchical">
                     <t>
-                        The "Detailed" structure is based on the schema and can be more readable
-                        for both humans and machines.  Having the structure organized this way makes
-                        associations between the errors more apparent.  For example, the fact that
-                        the missing "y" property and the extra "z" property both stem from the same
-                        location in the instance is not immediately obvious in the "Basic" structure.
-                        In a hierarchy, the correlation is more easily identified.
+                        The "Hierarchical" structure is a tree structure that follows the
+                        evaluation path during the validation process.  Typically, it will
+                        resemble the schema as if all referenced schemas were bundled in place
+                        of their associated by-reference keywords.
                     </t>
                     <t>
-                        The following rules govern the construction of the results object:
-                        <list>
-                            <t>
-                                All applicator keywords ("*Of", "$ref", "if"/"then"/"else", etc.) require
-                                a node.
-                            </t>
-                            <t>
-                                Nodes that have no children are removed.
-                            </t>
-                            <t>
-                                Nodes that have a single child are replaced by the child.
-                            </t>
-                        </list>
-                        Branch nodes do not require an error message or an annotation.
+                        All output units are included in this format.
+                    </t>
+                    <t>
+                        The location properties of the root output unit MAY be omitted.
```

Comment:
What does this mean exactly?  I would really rather not have to special-case any output unit, they should all always provide the same set of location information.

Reply:
For the `basic` output, results from the root schema are also an item in the list, which means that there is no need to have location properties (`evaluationPath`, `schemaLocation`, & `instanceLocation`) in the root output unit (and only the root).  Thus for `basic` it's just `valid` and `nested`.

This allowance permits these properties to also be omitted from the root of the `hierarchical` format.  The thinking behind this is that it aligns with `basic`, if that matters to implementors, but also that these location properties will _always_ be

```
evaluationPath: "" (empty pointer)
schemaLocation: "https://example.com/schema#"
instanceLocation: "" (empty pointer)
```

Having them isn't useful or really required.

Reply:
This feels like a design smell from trying to make two things that aren't really the same pretend like they are the same.